transactions/pseudo-random-generator/next_uint64.cdc

```diff
@@ -0,0 +1,15 @@
+import "PseudoRandomGenerator"
+
+/// Saves and links a .PRG resource in the signer's storage and public namespace
+///
+transaction(generationLength: Int) {
+    prepare(signer: AuthAccount) {
+        if let prg = signer.borrow<&PseudoRandomGenerator.PRG>(from: PseudoRandomGenerator.StoragePath) {
+            var i = 0
+            while i < generationLength {
+                prg.nextUInt64()
```

Comment:
We aren't able to return values from transactions, so a caller would need to advance the state of the PRG then query the generated values from a script. If the caller wanted to capture the generated values, they would need to be stored in some resource. I'm also realizing this would all be helpful context in this transaction's prefixed comment